manifests/eyaml.pp

```diff
@@ -41,42 +39,41 @@
       Exec['install puppetserver gem hiera-eyaml'],
     ]
 
-    if $::pe_server_version {
-      # PE 2015
-      $vendored_gem_creates = '/opt/puppetlabs/puppet/bin/eyaml'
-      $puppetserver_gem_creates = '/opt/puppetlabs/server/data/puppetserver/jruby-gems/bin/eyaml'
-    } else {
-      $vendored_gem_creates = '/opt/puppet/bin/eyaml'
-      $puppetserver_gem_creates = '/var/opt/lib/pe-puppet-server/jruby-gems/bin/eyaml'
-    }
-
     # The puppetserver gem wouldn't install the commandline util, so we do
-    # that here
-    #XXX Pre-puppet 4.0.0 version (PUP-1073)
+    # that here (PUP-1073)
     #BUG This can't actually update the gem version if already installed.
     if $eyaml_version and $eyaml_version =~ /^\d+\.\d+\.\d+$/ {
       $gem_flag = "--version ${eyaml_version}"
     } else {
       $gem_flag = undef
     }
-    #XXX Post-puppet 4.0.0
-    #package { 'hiera-eyaml command line':
-    #  ensure   => installed,
-    #  name     => 'hiera-eyaml',
-    #  provider => 'pe_gem',
-    #  source   => $gem_source,
-    #}
 
     exec { 'install ruby gem hiera-eyaml':
       command => "gem install hiera-eyaml ${gem_flag}",
-      creates => $vendored_gem_creates,
+      creates => '/opt/puppet/bin/eyaml',
```

Comment:
/opt/puppetlabs/puppet/bin/eyaml?  would need to run it but I don't think that path is correct